cmds/umount/umount_linux.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+	"golang.org/x/sys/unix"
+	"log"
+)
+
+const noFollow = 8
+
+var (
+	force = flag.Bool("f", false, "Force unmount")
+	lazy  = flag.Bool("l", false, "Lazy unmount")
+)
+
+func umount() {
+	var flags = noFollow
+	flag.Parse()
+	a := flag.Args()
+	if len(a) < 1 {
+		log.Fatalf("Usage: umount [-f] [-l] path")
+	}
+	if *force && *lazy {
+		log.Fatalf("force and lazy unmount cannot both be set")
+	}
+	path := a[0]
+	if *force {
+		flags |= unix.MNT_FORCE
+	}
+	if *lazy {
+		flags |= unix.MNT_DETACH
+	}
+	if err := unix.Unmount(path, flags); err != nil {
+		log.Fatalf("Umount :%s: flags %x: %v\n", path, flags, err)
```

Comment:
Return the error to the topmost level (the `main` function) and `log.Fatalf` there.

So this line becomes `return fmt.Errorf(...)`.